apisix/plugins/prometheus/exporter.lua

```diff
@@ -115,15 +116,20 @@ function _M.log(conf, ctx)
 
     local route_id = ""
     local balancer_ip = ctx.balancer_ip or ""
-    local service_id
+    local service_id = ""
```

Comment:
Better to use `service_id = vars.host` to be compatible

Reply:
Resolved since host is not related to service